fm-consumer@.service

```
@@ -1,3 +1,6 @@
+SPDX-FileCopyrightText: 2024 Red Hat, Inc
+SPDX-License-Identifier: GPL-2.0-or-later
```

This requires a prefix comment. Systemd unit files are INI files, and therefore can use `;` or `#` as a comment prefix. I would go for `#` since that's what I see most of the files in `/usr/lib/systemd/system` using.

```suggestion
# SPDX-FileCopyrightText: 2024 Red Hat, Inc
# SPDX-License-Identifier: GPL-2.0-or-later
```